google/ggrc-core
# GGRC-5824 Prevent mapping Person to any object

# Dependencies

This PR is `on hold` until the dependencies are merged:

- [x] #8307 
- [x] #8335

# Issue description

User has a possibility to Select People in Unified Mapper.

# Steps to test the changes

1. Log in to GGRC
2. Open All objects page
3. Open any tab, e.g. Regulation
4. Hover over tree view > gear icon> Map to this object
5. Select 'People' object type

**Actual Result:** 'People' object type is displayed in Unified mapper
**Expected Result:** User should not have a possibility to map a person via Unified mapper. 'People' object type should not be displayed in Unified mapper

# Solution description

Remove Person model from GGRC mappings config.

# Sanity checklist




# PR Review checklist
